### PR TITLE
Add rfc process docs changes expectations

### PR DESCRIPTION
**Pull request by darach**
An attempt to capture the relationship between work requiring RFCs and the
need for all significant changes to tremor to be well documented.

We are currently monitoring this pseudo-informally and tracking as/when needed
but this process should be written down over time.

### CODE REVIEW

File: rfc/index.md [hunk 1]
```diff
@@ -119,24 +122,24 @@ merged into the RFC repository as a markdown file. At that point the RFC is
...
+    strong consensus *against* that position outside of the sub-team. Sub-team
     members use their best judgment in taking this step, and the FCP itself
     ensures there is ample time and notification for stakeholders to push back
     if it is made prematurely.
     - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
       a *summary comment* trying to lay out the current state of the discussion
       and major tradeoffs/points of disagreement.
-    - Before actually entering FCP, *all* members of the subteam must sign off;
-    this is often the point at which many subteam members first review the RFC
+    - Before actually entering FCP, *all* members of the sub-team must sign off;
```
Comment (Licenser) [inline rfc/index.md:141]: Do we want to sign off from everyone? This makes things like vacations, illness, a process problem. Perhaps we can use `a majority and no objections`?

File: rfc/process/docs_changes.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+All RFCs are monitored by the documentation sub-team who will do an initial triage of new `needs-rfc` PRs within a week of submission. The result of triage be a decision on the documentation needs for the PR or RFC candidate.
+
+Generally speaking all user facing functions, interactions or behaviors requires
+both reference and practical `how to get jobs done` guides and be accompanied
+with functional tutorials and usage examples.
+
+## Since
+
+This is a new process document since 2023-Sep-22.
```
Comment (skoech) [inline rfc/process/docs_changes.md:24]: 2021. :D

File: rfc/index.md (hunk 1 above)
Comment (darach) [reply rfc/index.md:141]: This is consistent with other sub-teams - We probably want to reconsider that though ... so we can all have vacations!

File: rfc/process/docs_changes.md (hunk 2 above)
Comment (darach) [reply rfc/process/docs_changes.md:24]: I was hoping someone would find that 👁️ 🦅

**Review by Licenser — APPROVED** [review]

File: rfc/index.md (hunk 1 above)
Comment (Licenser) [reply rfc/index.md:141]: ja I was thinking about this in general (for all the teams) I'll drag it into a ticket

**Pull request merged**